test/alternator/test_tag.py

```diff
@@ -291,3 +292,53 @@ def test_tag_lsi_gsi(table_lsi_gsi):
         table_lsi_gsi.meta.client.tag_resource(ResourceArn=gsi_arn, Tags=tags)
     with pytest.raises(ClientError, match='ValidationException.*ResourceArn'):
         table_lsi_gsi.meta.client.tag_resource(ResourceArn=lsi_arn, Tags=tags)
+
+# Test that if we concurrently add tags A and B to a table, both survive.
+# If the process of adding tag A involved reading the current tags, adding
+# A and then over-writing the tags back, if we did this for A and B
+# concurrently the risk is that both would read the state before both changes.
+# To solve this, Scylla needs to serialize tag modification. This test
+# is designed to fail if this serialization is missing.  Reproduces #6389
+@pytest.mark.veryslow
```

Comment:
@avikivity I recall you were against introducing tests that are not run in CI by default.

If the test takes only a few seconds, I'm for enabling it by default. There are other longer tests in the suite, this one will run concurrently with them so I think it's fine.

Reply:
The entire Alternator test suite, 711 tests in 18,000 lines of code, takes today 28 second. So "a few seconds" isn't exactly fine.

I think we can reconsider this later. We talked in the past about having an option that runs longer tests only in some CI runs but every time a developer runs tests on their laptop, maybe we can use that idea.

Philosophically, there are several levels of testing, with increasing rigor:

1. I didn't bother to test, the code compiled!
2. Believe me, I tested myself.
3. Here is the test I ran, once. You can see what I forgot to test, and can also (maybe) run the same test code again a year from now.
4. Here is the test I ran, let's run it every day.

In this case I went with level 3, which is perhaps not as great as level 4, but still better than level 2 or 1 :-)

Reply:
I was referring to the entire "test.py suite", developers most often just run the whole test.py instead of specific sub-suites. Anyway, generally I agree with you.